DEVELOPING.rst

```diff
@@ -22,6 +25,23 @@ integration purposes. A usage example:
    $ firefox http://0.0.0.0:5000/
    $ docker-compose -f docker-compose-dev.yml down
 
+When working on UI packages that have JavaScript and CSS files, you can have
+"live editing" by running the following command on a new terminal:
+
+.. code-block:: console
+
+   $ docker exec -i -t opendatacernch_web_1 cernopendata webpack run start
+
+Keep in mind that you need to recreate the ``package.json`` when adding or
+removing dependencies:
+
+.. code-block:: console
+
+   $ docker exec -i -t opendatacernch_web_1 cernopendata webpack clean create
```

Comment:
I wonder whether we could move this under `Development tips` as a new section "frontend development" or similar so we keep the installation instructions simple, what do you think?